recipes-app/node-red/node-red_2.0.6.bb

```diff
@@ -13,6 +13,8 @@ inherit npm
 
 DESCRIPTION = "A visual tool for wiring the Internet of Things"
 
+DEBIAN_BUILD_DEPENDS =. "libnode-dev,"
+
```

Comment:
For convenience reasons, I'm inclined to add this npm.bblcass, even if not all npm packages may contain native elements that require this. You could make that dep conditional, only adding it if NPM_REBUILD is set.